src/main/java/racing/ResultView.java

```diff
@@ -0,0 +1,36 @@
+package racing;
+
+public class ResultView {
+    private RacingVO racingVO;
+    private RacingGame racingGame;
+
+    public ResultView(RacingVO racingVO, RacingGame racingGame) {
+        this.racingVO = racingVO;
+        this.racingGame = racingGame;
+    }
+
+    public void view() {
+        int count = racingVO.getCount();
+        System.out.println("실행결과");
+        for(int i = 0; i < count; ++i) {
+            viewResult(racingGame.move());
+        }
+    }
+
+    public void viewResult(int[] carPositions) {
+        int length = carPositions.length;
+        for(int i = 0; i < length; i++) {
+            int carPosition = carPositions[i];
+            System.out.println(printCar(carPosition));
+        }
+        System.out.println("###############################################");
+    }
+
+    public String printCar(int carPosition) {
+        StringBuffer sb = new StringBuffer();
+        for(int i = 0; i < carPosition; ++i) {
+            sb.append("-");
+        }
+        return sb.toString();
```

Comment:
```suggestion
    public void printResult(int[] positions) {
        Arrays.stream(positions)
                .mapToObj(position -> CAR.repeat(position))
                .forEach(System.out::println);

        printBorderLine();
    }
```

Stream API를 활용하여 반복문을 내부 라이브러리에 위임함으로써 무엇을 연산할지를 드러내는 것은 어떨까요